.gitignore

```diff
@@ -48,4 +48,7 @@ src/config.h
 
 # Ignore the build directory
 build/*
-!build/.gitkeep
+builddir/*
+
```

Comment:
Why did you remove .gitkeep and add builddir to this?

Reply:
I removed the .gitkeep because Meson doesn't require the build folder to exist beforehand, but there's no other advantage besides having one less folder.
I added builddir since it's the default on the Meson documentation and the one I have been using. I can remove it.

Reply:
Can you send me a link to the Meson documentation?

Reply:
[Mesonbuild documentation - configure build directory](https://mesonbuild.com/Running-Meson.html#configuring-the-build-directory)

Reply:
I think we should make it ignore both `build` and `builddir`.

Reply:
Simply do not ignore anything.

Since Meson 0.57.0, it creates `build/.gitignore` or `builddir/.gitignore` but also `builddir-release/.gitignore`, `builddir-debug/.gitignore`, `_build/.gitignore`, or anything else you might need.

It creates it with the contents `*` so as to ignore all generated files, you don't need to do anything yourself to set this up.